.travis.yml

```diff
@@ -95,11 +95,10 @@ script:
   # test parallel version
   - python setup.py build_ext --with-tests
   - python setup.py test -v
+  - python setup.py benchmark -v
 
   # test series version
-  - export FOAM_WITH_TBB=0
-  - export FOAM_WITH_XSIMD=0
-  - export XTENSOR_WITH_TBB=0
-  - export XTENSOR_WITH_XSIMD=0
+  - export BUILD_SERIAL_FOAM=1
```

Comment:
@RobertRosca I added one environmental variable to allow switch to serial version.